synapse/tools/autodoc.py

```diff
@@ -220,5 +260,67 @@ def main(argv, outp=None):
         outp.printf(rst.getRstText())
         return 0
 
+    if opts.configable_opts:
```

Comment:
what was more impressive was that EVERYTHING except the docker tree was directly importable without blowing up :)